### ISSUE TITLE
Why custom BatchNormLayer?

### ISSUE THREAD

**Issue by sziraqui** [auroua/InsightFace_TF#89]
Thanks for your TF implementation of ArcFace related models. I am trying to port this into tfjs-node but I am stuck because of custom layers you have implemented in `nets/L_Resnet_E_IR_fix_issue9.py`.
Is your BatchNormLayer different than TensorFlow's BatchNormalisation layer? If yes, could you shed some light on their differences?
